tests/update_images/testdata/update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+az vm image list -f RHEL -l eastus --all > azure_list_images.json
```

Comment:
To use the Azure client from your bash script like this you need to run `az login` first.
I'm not sure about the gcloud client but it seems that the last test run failed silently on both and pushed empty json files: https://github.com/redhatcloudx/rhelocator/pull/136/files#diff-ce5b5932601b8d4b43b766097e356bbc1a6da18293462b528f6a8f55fce95f44

Reply:
K seems like gcloud needs an account setup as well: https://github.com/redhatcloudx/rhelocator/actions/runs/3531503702/jobs/5924749016#step:3:13

Reply:
Brings up a good point though: From what it seems, the failure of the first two client runs are not propagated to the github actions because the last command (the aws cli) succeeds and the bash script exit code is set to 0. Would be nice if we could safeguard this so that the run fails if any of the cli commands fail.

Reply:
> Brings up a good point though: From what it seems, the failure of the first two client runs are not propagated to the github actions because the last command (the aws cli) succeeds and the bash script exit code is set to 0. Would be nice if we could safeguard this so that the run fails if any of the cli commands fail.

oh, this message just popped up now! :smile: But that makes sense to me aswell!!

Reply:
Hm, I would assume that there is a work around for this.. at least for gcloud. I mean we already create the credential JSON file so we should be able to authenticate without another login step.. for Azure however I'm unsure. It would be worth reading up on it before manually login in again. Another option would be to check out if there are official Github Actions for that.

Reply:
> Another option would be to check out if there are official Github Actions for that.

Thanks that was a good hint [[azure-login-action](https://github.com/marketplace/actions/azure-login)]. :smiley: Unfortunately, I have not yet got it to run (azure part) and for now I just commented it out. Once I have changed/written the tests for the azure data, I would try to activate it. Would that be ok? :see_no_evil: Or do you have an idea how to fix it? :/

Google and AWS are now working so far #145 :fireworks: